make_image.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
```

Comment:
Are you using any bashisms? Otherwise might be more portable with `#!/bin/sh`

And if it really has to be Bash it is better to use `#!/usr/bin/env bash`

Reply:
Yes I do, I use trap to register a hook on script exit to cleanup, and I think it is bash specific